locations/spiders/bricomarche_pl.py

```
@@ -6,14 +6,16 @@
 from locations.categories import Categories, apply_category
 from locations.items import Feature
 from locations.json_blob_spider import JSONBlobSpider
+from locations.playwright_spider import PlaywrightSpider
+from locations.settings import DEFAULT_PLAYWRIGHT_SETTINGS
 
 
-class BricomarchePLSpider(JSONBlobSpider):
+class BricomarchePLSpider(JSONBlobSpider, PlaywrightSpider):
     name = "bricomarche_pl"
     item_attributes = {"brand": "Bricomarché", "brand_wikidata": "Q2925147"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)